packages/runtime/container-runtime/package.json

```diff
@@ -116,40 +116,38 @@
   "typeValidation": {
     "version": "0.59.1000",
     "broken": {
```

Comment:
as a test/example I unnested from the version here. the results are as expected.